collectorApp.php

```diff
@@ -1,5 +1,19 @@
 <?php
 
+$connectionString = 'mysql:host=db; dbname=planetdatabase';
+$dbUsername = 'root';
+$dbPassword = 'password';
+$db = new PDO($connectionString, $dbUsername, $dbPassword);
+
```

Comment:
might be worth adding the below line

$db->setAttribute(PDO::ATTR_DEFAULT_FETCH_MODE, PDO::FETCH_ASSOC);